docs/trouble_shooting.md

```diff
@@ -48,4 +48,4 @@ This guide (based on minikube and others should be similar) explains general inf
    ```
 
 2. Check docker version.
-   kind v0.3.0 can work with docker 18.06.0 on ubuntu.
+   You may hit errors with kind v0.3.0 and docker 18.09.6 on ubuntu. Use docker version < 18.09.6.
```

Comment:
Please get a conclusion at https://github.com/kubernetes-sigs/kind/issues/567#issuecomment-496097511 first before updating the document.

Reply:
No iptable chain will cause this issue. 
moby/moby#1871 can help on it.